pkg/storaged/resize.jsx

```diff
@@ -281,18 +338,30 @@ export function grow_dialog(client, lvol_or_part, info, to_fit) {
     }
 
     let grow_size;
-    let size_fields = [];
+    const size_fields = [];
     if (!to_fit) {
-        size_fields = [
+        if ((has_subvols || lvol_or_part.Layout == "linear") && pvs_as_spaces.length > 1)
+            size_fields.push(
+                SelectSpaces("pvs", _("Physical Volumes"),
+                             {
+                                 spaces: pvs_as_spaces,
+                                 value: initial_pvs,
+                                 min_selected: subvols.length,
+                                 validate: val => {
+                                     if (has_subvols && subvols.length != val.length)
+                                         return cockpit.format(_("Exactly $0 physical volumes must be selected"),
+                                                               subvols.length);
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17226-20231026-132112-61c336d9-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)